project/jpype_java/test/org/jpype/pkg/JPypePackageNGTest.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * To change this license header, choose License Headers in Project Properties.
+ * To change this template file, choose Tools | Templates
+ * and open the template in the editor.
+ */
+package org.jpype.pkg;
+
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.Map;
+import static org.testng.Assert.*;
+import org.testng.annotations.Test;
+
+/**
+ *
+ * @author nelson85
+ */
+public class JPypePackageNGTest
+{
+
+  public JPypePackageNGTest()
+  {
+  }
+
+  @Test
+  public void testGetPackage()
+  {
+    assertTrue(JPypePackageManager.isPackage("java.lang"));
+    JPypePackage pkg = new JPypePackage("java.lang", JPypePackageManager.getContentMap("java.lang"));
+    for (Map.Entry<String, Path> e : pkg.contents.entrySet())
+    {
+      JPypePackage.isPublic(e.getValue());
```

Comment:
is that an assertion?

Reply:
It is just a check for robustness.  The goal of the statement was to make sure nothing would generate an exception while reading a whole bunch of constant pools.  Unfortunately there is no way to know in jar files if you are actually synced up as the only guaranteed structure is the magic number.

I guess we could toss a bunch of known values in, but it would be better to use a structure we control as java/lang can have anything in it.